de.metas.async/src/main/java/de/metas/async/api/IQueueDAO.java

```diff
@@ -136,4 +136,6 @@
 	 * @return
 	 */
 	boolean isWorkpackageProcessorEnabled(Class<? extends IWorkpackageProcessor> packageProcessorClass);
+
+	<T> List<T> retrieveAllItems(I_C_Queue_WorkPackage workPackage, Class<T> clazz);
```

Comment:
pls add this method new the other retrieveItems-methods and add a javadoc to describe what "all" means.
Usually, with "all" in the methodname, i would expect it to also return inactive records.
Also, please describe when and why a dev should call this method